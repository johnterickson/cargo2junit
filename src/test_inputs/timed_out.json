{ "type": "suite", "event": "started", "test_count": 1 }
{ "type": "test", "event": "started", "name": "tests::long_execution_time" }
{ "type": "test", "name": "tests::long_execution_time", "event": "failed", "exec_time": 1.0120183, "reason": "time limit exceeded" }
{ "type": "suite", "event": "failed", "passed": 0, "failed": 1, "allowed_fail": 0, "ignored": 0, "measured": 0, "filtered_out": 10, "exec_time": 1.0151028 }
