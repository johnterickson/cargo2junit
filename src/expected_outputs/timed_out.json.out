<?xml version="1.0" encoding="utf-8"?>
<testsuites>
  <testsuite id="0" name="cargo test #0" package="testsuite/cargo test #0" tests="1" errors="0" failures="1" hostname="localhost" timestamp="2021-03-06T23:56:08.701117400+00:00" time="1.0120183">
    <testcase name="long_execution_time" classname="tests" time="1.0120183">
      <failure type="cargo test" message="time limit exceeded" />
    </testcase>
  </testsuite>
</testsuites>